attributes/master.rb

```diff
@@ -135,6 +135,11 @@
   master['use_system_accounts'] = true
 
   #
+  # The groups of user under which Jenkins is running. Works for runit only.
+  #
+  master['groups'] = [node['jenkins']['master']['group']]
```

Comment:
I'm not the originator of the pull request, but for our case, we would like to set the -u and -U switches in the chpst command to "jenkins:jenkins:docker". This runs the process as the "jenkins" user with groups "jenkins" and "docker". I think a separate variable is useful to allow the process be ran with multiple group permissions.

The name "master['groups']" may not be a good, descriptive name for the intent of the variable.